{"uuid":"b3c50bec-92a0-4022-8d60-a62c37525d08","historyId":"fc5d9fbe1813873b7b36b12888de97e4","fullName":"died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest01","labels":[{"name":"package","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"testClass","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"testMethod","value":"ForgotPassCaseTest01"},{"name":"parentSuite","value":"Login"},{"name":"suite","value":"Test_03"},{"name":"subSuite","value":"died_procesos_generales.pgForgotPassTest.ForgotPassTest"},{"name":"host","value":"ALQ000002139"},{"name":"thread","value":"13544@ALQ000002139.main(1)"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Forgot Tests"}],"links":[],"name":"ForgotPassCaseTest01","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Los datos han sido actualizados satisfactoriamente.] but found []","trace":"java.lang.AssertionError: expected [Los datos han sido actualizados satisfactoriamente.] but found []\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage02_aroundBody8(ForgotPassPage.java:139)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage$AjcClosure9.run(ForgotPassPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage02(ForgotPassPage.java:139)\r\n\tat died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest01(ForgotPassTest.java:53)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1549654245587,"stop":1549654286768,"steps":[{"name":"Open N12 Step...","status":"passed","stage":"finished","start":1549654245623,"stop":1549654249853},{"name":"metodo de recuperar contraseña con email","status":"passed","stage":"finished","start":1549654250364,"stop":1549654259440,"parameters":[{"name":"row","value":"<xml-fragment r=\"11\" spans=\"1:10\" x14ac:dyDescent=\"0.25\" xmlns:mc=\"http://schemas.openxmlformats.org/markup-compatibility/2006\" xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\" xmlns:x14ac=\"http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac\" xmlns:main=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\">\r\n  <main:c r=\"A11\" s=\"5\">\r\n    <main:v>1</main:v>\r\n  </main:c>\r\n  <main:c r=\"B11\" s=\"1\" t=\"s\">\r\n    <main:v>29</main:v>\r\n  </main:c>\r\n  <main:c r=\"C11\" s=\"10\" t=\"s\">\r\n    <main:v>37</main:v>\r\n  </main:c>\r\n  <main:c r=\"D11\" s=\"19\" t=\"s\">\r\n    <main:v>38</main:v>\r\n  </main:c>\r\n  <main:c r=\"E11\" s=\"2\"/>\r\n  <main:c r=\"F11\" s=\"2\" t=\"s\">\r\n    <main:v>31</main:v>\r\n  </main:c>\r\n  <main:c r=\"G11\" s=\"2\"/>\r\n  <main:c r=\"H11\" s=\"2\" t=\"s\">\r\n    <main:v>34</main:v>\r\n  </main:c>\r\n  <main:c r=\"I11\" s=\"1\" t=\"s\">\r\n    <main:v>18</main:v>\r\n  </main:c>\r\n  <main:c r=\"J11\" s=\"18\" t=\"s\">\r\n    <main:v>36</main:v>\r\n  </main:c>\r\n</xml-fragment>"}]},{"name":"Verify username: {0} step...","status":"passed","stage":"finished","start":1549654259998,"stop":1549654263340,"parameters":[{"name":"expectedUserNameMessage","value":" En unos momentos le llegará un link donde podrá cambiar su contraseña, por favor revise su correo electrónico. También revise su bandeja de correos no deseados y/o SPAM."}]},{"name":"metodo para ingresar al email","status":"passed","stage":"finished","start":1549654269075,"stop":1549654286129,"parameters":[{"name":"row","value":"<xml-fragment r=\"11\" spans=\"1:10\" x14ac:dyDescent=\"0.25\" xmlns:mc=\"http://schemas.openxmlformats.org/markup-compatibility/2006\" xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\" xmlns:x14ac=\"http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac\" xmlns:main=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\">\r\n  <main:c r=\"A11\" s=\"5\">\r\n    <main:v>1</main:v>\r\n  </main:c>\r\n  <main:c r=\"B11\" s=\"1\" t=\"s\">\r\n    <main:v>29</main:v>\r\n  </main:c>\r\n  <main:c r=\"C11\" s=\"10\" t=\"s\">\r\n    <main:v>37</main:v>\r\n  </main:c>\r\n  <main:c r=\"D11\" s=\"19\" t=\"s\">\r\n    <main:v>38</main:v>\r\n  </main:c>\r\n  <main:c r=\"E11\" s=\"2\"/>\r\n  <main:c r=\"F11\" s=\"2\" t=\"s\">\r\n    <main:v>31</main:v>\r\n  </main:c>\r\n  <main:c r=\"G11\" s=\"2\"/>\r\n  <main:c r=\"H11\" s=\"2\" t=\"s\">\r\n    <main:v>34</main:v>\r\n  </main:c>\r\n  <main:c r=\"I11\" s=\"1\" t=\"s\">\r\n    <main:v>18</main:v>\r\n  </main:c>\r\n  <main:c r=\"J11\" s=\"18\" t=\"s\">\r\n    <main:v>36</main:v>\r\n  </main:c>\r\n</xml-fragment>"}]},{"name":"Verify username: {0} step...","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Los datos han sido actualizados satisfactoriamente.] but found []","trace":"java.lang.AssertionError: expected [Los datos han sido actualizados satisfactoriamente.] but found []\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage02_aroundBody8(ForgotPassPage.java:139)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage$AjcClosure9.run(ForgotPassPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_procesos_generales.pgForgotPassPage.ForgotPassPage.verifyMessage02(ForgotPassPage.java:139)\r\n\tat died_procesos_generales.pgForgotPassTest.ForgotPassTest.ForgotPassCaseTest01(ForgotPassTest.java:53)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1549654286131,"stop":1549654286177,"parameters":[{"name":"expectedUserNameMessage","value":"Los datos han sido actualizados satisfactoriamente."}]}],"parameters":[]}